pkgs/by-name/za/zapzap/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zapzap";
+  version = "4.5.5.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "zapzap-linux";
+    repo = "zapzap";
+    rev = version;
+    hash = "sha256-8IeFGTI+5kbeFGqH5DpHCY8pqzGhE48hPCEIKIe7jAM=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    dbus-python
+    pyqt6
+    pyqt6-webengine
+    pyqt6-sip
+  ];
+
+  prePatch = "export HOME=$(mktemp -d)";
+
+  postInstall = ''
+    install -Dm555 share/applications/com.rtosta.zapzap.desktop -t $out/share/applications/
+    install -Dm555 share/icons/com.rtosta.zapzap.svg -t $out/share/icons/hicolor/scalable/
+  '';
+
```

```suggestion

  # has no tests
  doCheck = false;

```

```
zapzap> Ran 0 tests in 0.000s
zapzap>
zapzap> OK
zapzap> Finished executing setuptoolsCheckPhase
```

We don't need to try to run tests if there are none